openapi/consumed/api_remote_content.yaml

```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.1
+info:
+  title: Remote Content - Server implementation
+  version: 9.7.0
```

Comment:
maybe this is not really 9.7.0... we are at 1.0.0 I think... for this remote content apis.